### ISSUE TITLE
Texture files are not loading

### ISSUE THREAD

**Issue by HARADA-SSV** [Rahala/SweetConcrete#5]
Actually, I don't know what's going on, sometimes I have to restart the luncher a second or even fifth time for the textures of the plates to be loaded. Interestingly, the **F3+T** shortcut, which is to reload textures already in the enabled game, does not work on them. I present the picture below

### Technical data
Minecraft 1.16.5 Forge 1.16.5 - 36.2.19
Sweetconcrete-1.16.5-1.7.2

![2021-11-24_23 27 39](https://user-images.githubusercontent.com/79222064/143321246-216522dc-bc0a-4818-bc46-40c5ddd60720.png)

**Comment by HARADA-SSV** [comment]
![2021-11-24_23 36 14](https://user-images.githubusercontent.com/79222064/143321454-c444c900-17a0-4e41-81e8-cbe13b1cd275.png)

**Comment by Angela** [comment]
@HARADA-SSV Thank you for reporting this bug. I'm sorry that you are experiencing this problem.

If I understand the problem correctly, sometimes after launching the game, when you place a concrete sign in the world, it shows with the oak sign texture rather than the correct concrete texture. Reloading textures does not help, but sometimes launching the game again from the launcher will cause the signs to have the correct texture?

Does this happen with any other block than the signs, e.g. slabs, ladders, etc.?

Can you confirm if you have any other mods installed in addition to Forge/Sweet Concrete?

**Comment by HARADA-SSV** [comment]
Yes that's right Reloading textures doesn't help, but sometimes restarting the game solves the problem. other blocks are correctly displayed and do not have texture converted to oak. Below is a list of modifications that I have installed.
![Zrzut ekranu 2021-11-25 025029](https://user-images.githubusercontent.com/79222064/143359367-45e9a0b8-2793-4c48-8a42-cab92ff6ed3a.png)
